scripts/validate

```diff
@@ -32,5 +32,3 @@ if [ -n "$DIRTY" ]; then
     git diff
     exit 1
 fi
-
-"${GO}" test -v ./pkg/...
```

Comment:
By removing this, we may want to add a new target to the Makefile for local use.

Reply:
Add a new "make test" to the Makefile